src/Components/Auth/UnAuthenticatedRoute.jsx

```diff
@@ -0,0 +1,9 @@
+import { Navigate, Outlet, useLocation } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+
+const UnAuthenticatedRoute = () => {
```

Comment:
Is there any need for Unauthenticated route.
Because everyone route that is not enclosed with PrivateRoute will be treated as Unauthenticated route by default.